api/src/wsmanager.py

```diff
@@ -18,6 +19,10 @@ def is_valid_uuid(uuid_string):
     return val.hex == uuid_string.replace('-', '')
 
 
+def ignore_none(items: List[Tuple[str, Any]]) -> dict:
```

Comment:
You'll have to add the correct imports at the top
```suggestion
KT = TypeVar('KT', bound=Hashable)
VT = TypeVar("VT")
def ignore_none(items: List[Tuple[KT, VT]]) -> Dict[KT, VT]:
```